configlib/config.go

```diff
@@ -204,3 +204,13 @@ func remove(ymlfile string, packageName string) error {
 	}
 	return nil
 }
+
+// SetCustomizations ... set ENV values in Rsettings
+func SetCustomizations(cfg PkgrConfig, rSettings *rcmd.RSettings) {
```

Comment:
if we're going to be just modifying the rSettings, why not have setCustomizations be a method on it?

```
rs.SetCustomizations(cfg)
```

Reply:
alternatively, if we keep this a function, I still vastly prefer the functional style then, give an object, return a new object, and in that case I would also change the order of args

```
rs = SetCustomizations(rs, cfg)
```

Reply:
OK, I'll look into creating a method